src/features/cluster/kube-object-status-icon/extension-api/reactively-hide-kube-object-status.test.tsx

```diff
@@ -71,24 +72,16 @@ describe("reactively hide kube object status", () => {
     builder.quit();
   });
 
-  it("does not show the kube object status", () => {
-    const actual = rendered.baseElement.querySelectorAll(
-      ".KubeObjectStatusIcon",
-    );
-
-    expect(actual).toHaveLength(0);
+  it("does not show the kube object status", async () => {
+    await waitFor(() => expect(rendered.baseElement.querySelectorAll(".KubeObjectStatusIcon")).toHaveLength(0));
```

Comment:
This is slow unit test. In unit test we should always know if the item is there or not.